L1Trigger/L1THGCal/src/backend/HGCalHistoSeedingImpl.cc

```diff
@@ -92,18 +114,23 @@ HGCalHistoSeedingImpl::Histogram HGCalHistoSeedingImpl::fillSmoothPhiHistoCluste
           if (binToSumRight >= int(nBinsPhiHisto_))
             binToSumRight -= nBinsPhiHisto_;
 
-          content += histoClusters.at({{z_side, bin_R, binToSumLeft}}) / pow(2, bin_phi2);   // quadratic kernel
-          content += histoClusters.at({{z_side, bin_R, binToSumRight}}) / pow(2, bin_phi2);  // quadratic kernel
+          content += histoClusters.at({{z_side, bin_R, binToSumLeft}}).sumMipPt / pow(2, bin_phi2);   // quadratic kernel
+          content += histoClusters.at({{z_side, bin_R, binToSumRight}}).sumMipPt / pow(2, bin_phi2);  // quadratic kernel
         }
+      
+        auto & bin =  histoSumPhiClusters[{{z_side, bin_R, bin_phi}}];
+        bin.sumMipPt = content / area;
+        bin.weighted_x = bin_orig.weighted_x;
+        bin.weighted_y = bin_orig.weighted_y;
 
-        histoSumPhiClusters[{{z_side, bin_R, bin_phi}}] = content / area;
       }
     }
   }
 
-  return histoSumPhiClusters;
+  return histoClusters;
```

Comment:
`histoSumPhiClusters` should be returned, otherwise the returned histo won't be the smoothed one